### ISSUE TITLE
Problem with (new?) @live.com accounts?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#34]
```
What steps will reproduce the problem?
1. Create a new account on Live (go to www.live.com, click on Sign In, and
then on Sign Up, then follow the instructions)
2. Configure that account on Pidgin using Pecan
3. Try to login

What is the expected output? What do you see instead?

It tries to connect, but can't (taking a long time trying). Debug log to
follow.

What version of the product are you using? On what operating system?

Pecan 0.11, Pidgin 2.4.1 (also tried on 2.4.0, yesterday), Windows XP.

Please provide any additional information below.

1- I can use Pecan for my older MSN account, which is @gmail.com .
2- I can use the standard Pidgin MSN (also Kopete on Linux) for my new
@live.com account.

Debug log attached.

```

Original issue reported on code.google.com by `pedro.ti...@gmail.com` on 1 Apr 2008 at 9:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `pedro.ti...@gmail.com` on 1 Apr 2008 at 9:43

Attachments:
- [purple-debug.log](https://storage.googleapis.com/google-code-attachments/msn-pecan/issue-34/comment-1/purple-debug.log)

**Comment by GoogleCodeExporter** [comment]
```
It seems to be working now, and I didn't change anything, or installed new 
versions
of anything (including Pidgin and Pecan). Weird. Anyway, I think you can close 
this.

```

Original comment by `pedro.ti...@gmail.com` on 16 Apr 2008 at 9:38

**Comment by GoogleCodeExporter** [comment]
```
There seems to be a log in issue that's already fixed and will be in 0.0.12.
Hopefully we won't have to re-open this.
```

Original comment by `felipe.contreras` on 29 Apr 2008 at 12:52
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Er, marking as fixed... can you verify that indeed it's working fine?
```

Original comment by `felipe.contreras` on 5 Feb 2010 at 2:49
- Changed state: **Fixed**
